lib/src/HttpUtils.cc

```diff
@@ -603,4 +603,28 @@ ContentType parseContentType(const string_view &contentType)
     return iter->second;
 }
 
+FileType parseFileType(const std::string &fileExtension)
```

Comment:
I think the parameter type should be string_view.

Reply:
I have already applied this suggestion. (https://github.com/an-tao/drogon/pull/803/commits/814d91a612fc878d7ad773bf6a3dbc0af09a4be5)